typescript-getting-started/functions/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "functions",
+  "description": "Cloud Functions for Firebase",
+  "dependencies": {
+    "firebase-admin": "~4.2.1",
+    "firebase-functions": "^0.5.7"
+  },
+  "devDependencies": {
+    "typescript": "^2.3.2"
+  },
+  "scripts": {
+    "build": "tsc",
+    "watch": "tsc --watch",
+    "deploy": "tsc && firebase deploy --only functions"
+  },
+  "main": "build/index.js",
```

Comment:
I'm surprised this works; it seems like a TypeScript bug. I think that if you had `src/index.ts` and `spec/index.spec.ts` you'll end up getting `build/src/index.js` build/spec/index.js`

Reply:
well, this example doesn't have tests yet, I'd be inclined not to put output for running tests in the build directory -- for example: could be spec/src and spec/build.  In any case, it works as is, and we can circle back and add tests later

Reply:
The concern is whether the production code would break simply by adding a test file.